src/Sentry.Unity.Native/SentryNativeBridge.cs

```diff
@@ -29,6 +30,40 @@ public static void Init(SentryUnityOptions options)
                 sentry_options_set_release(cOptions, options.Release);
             }
 
+            if (options.Environment is not null)
```

Comment:
Was it not possible to add the init code in the main function of the Windows executable instead of initializing here?
This approach is different than Android and iOS and would require further clarification on the docs. i.e:

> In order to provide native crash support, the Sentry SDK for Unity includes platform-specific (that is, Native) SDKs, such as [Android](https://docs.sentry.io/platforms/android/), [Apple](https://docs.sentry.io/platforms/apple/guides/ios/), and [Native](https://docs.sentry.io/platforms/native/).
> 
> By default, we initialize the native SDKs before the Unity engine itself. That means it also runs before the C# layer that relies on [BeforeSceneLoad RuntimeInitializeOnLoadMethodAttribute](https://docs.unity3d.com/ScriptReference/RuntimeInitializeLoadType.BeforeSceneLoad.html) to get initialized. This allows us to capture bugs/crashes of the engine itself, any native plugin, and anything written in C#. The setup for the native SDKs happens during build time, which is why we rely on the options being set in the Sentry editor configuration window and saved to Assets/Resources/Sentry/SentryOptions.asset. To provide a way to modify options programmatically, we've added ScriptableOptionsConfiguration to the Options Config tab in the Sentry editor window.

From: https://docs.sentry.io/platforms/unity/configuration/options/

Reply:
as discussed, that would be the next PR